### ISSUE TITLE
Filter for familiar drops all rows with data from WorkSocialPsychology

### ISSUE THREAD

**Issue by rcalinjageman** [seanrife/TM-RRR#8]
The filter to exclude those familiar with the study drops all rows that have data from the data file for WorkSocialPsychology:

   df <- df[df$Familiar==0,]

That is, the data file has all 1s or NAs, no 0s.  With no rows left, the rest of the processing for that lab fails.  Is there an error in this filter?

**Comment by seanrife** [comment]
Yes, this seems to be an issue with a number of labs. Removed (at least for now - see https://github.com/seanrife/TM-RRR/commit/e0fef06f552eba4058355c2837a1a509420d0c6f). I think it's quite possible that - at least in some places - TMT is widely known (definitely something to discuss).
